specification/portal/resource-manager/Microsoft.Portal/preview/2019-01-01-preview/tenantConfiguration.json

```diff
@@ -0,0 +1,204 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-01-01-preview",
+    "title": "portal",
+    "description": "Manage tenant configuration.",
+    "x-ms-code-generation-settings": {
+      "name": "portalClient"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "paths": {
```

Comment:
>paths [](start = 3, length = 5)

Should probably support GET /providers/Microsoft.Portal/tenantConfiguration so users can discover the singleton easily.

Reply:
as I mentioned above - there could be other configurations in the future, so this path will be used for configuration list

Reply:
Even with the singleton right now, you should have a call to list that returns just the one. Otherwise, how would a user know that "default" is the one that exists?

---
In reply to: [483794106](https://github.com/Azure/azure-rest-api-specs/pull/10603#discussion_r483794106) [](ancestors = 483794106)